build/Build.cs

```diff
@@ -11,7 +9,6 @@
 using static Nuke.Common.Tools.DotNet.DotNetTasks;
 using Nuke.Common.Tools.OctoVersion;
 
-[CheckBuildProjectConfigurations]
```

Comment:
This has since been integrated into the `SolutionAttribute`.
https://github.com/nuke-build/nuke/blob/4a24cd4e78d43b59deb8aa2033c696c8ceb51dd6/source/Nuke.Common/Execution/CheckBuildProjectConfigurationsAttribute.cs#L18-L21